common/src/nfts.rs

```diff
@@ -17,6 +17,9 @@ pub trait NFTs {
     /// How the NFT series id is represented internally.
     type NFTSeriesId: Parameter + Copy + Default;
 
+    /// TODO!
+    type Protocol: Parameter + Copy + Default;
+
```

Comment:
I discussed a bit with @ybensacq, I really don't think this pallet is the place to manage "protocols". We probably want to drop this for now

Reply:
Yeah, I saw the conversation. I was a bit worried that If I didn't implement the protocol thing, that someone would definitely ask after the merge where that parameter is and why it is missing xd 

So I should just drop it but keep the is_capsule flag?

Reply:
I think so, worth checking with @ybensacq

Reply:
Protocol is now removed and per @ybensacq instructions is_capsule is still left.